deploy/clowdapp.yaml

```diff
@@ -168,6 +184,18 @@ parameters:
   - description: Image name
     name: IMAGE
     value: quay.io/cloudservices/provisioning-backend
+  - description: Logging level (trace, debug, info, warn, error, fatal, panic)
+    name: LOGGING_LEVEL
+    value: "trace"
+  - description: Postgres driver logging level (trace, debug, info, warn, error, fatal, panic)
+    name: DATABASE_LOGGING_LEVEL
+    value: "trace"
```

Comment:
This will enable trace level for stage as well... I believe we should set these to "resonable" values and set the different values per environment in app-interface

Reply:
Yeah but I don't know how to do that now, so I was hoping we can live with increased verbosity for some time.

Reply:
Oki, I'm fine with that. If we can set no colors only in test env I'm ok. Or i can set levels to traces by parameters for ephemeral only if we agree to it

Reply:
So it looks like I figured out how to set ephemeral variables (through app-interface).